src/scripts/quiz.js

```diff
@@ -0,0 +1,13 @@
+var a = document.querySelector(".quiz-cl");
+
+let b = document.querySelector(".btnText");
+
+let c = document.querySelector('.bt2');
+
+b.addEventListener('click', ab());
```

Comment:
for JS using id selector will be better

Reply:
it's an unnecessary file I forgot to delete it.